src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -985,11 +994,13 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
 
                             for (int i = 0; i < overridingParameters.Length; i++)
                             {
-                                var overridenParameterType = overriddenParameters[i].TypeWithAnnotations;
+                                var overriddenParameterType = overriddenParameters[i].TypeWithAnnotations;
                                 var overridingParameterType = overridingParameters[i].TypeWithAnnotations;
-                                if (!overridingParameterType.Equals(overridenParameterType,
-                                                                         TypeCompareKind.AllIgnoreOptions & ~(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes)) &&
-                                    overridingParameterType.Equals(overridenParameterType, TypeCompareKind.AllIgnoreOptions))
+                                if (!isValidNullableOverride(
```

Comment:
>isValidNullableOverride [](start = 37, length = 23)

It feels like parameter types and return types should be treated differently, but it doesn't look like there is an obvious difference in how we call the helper.